src/widgetastic_patternfly/__init__.py

```diff
@@ -2249,3 +2249,82 @@ class AggregateStatusMiniCard(AggregateStatusCard):
         'and h2[contains(@class, "card-pf-title")]'
         '//span[normalize-space(following::text())={@name|quote}]]'
     )
+
+
+class Kebab(Widget):
+    """Patternfly Kebab menu widget
+
+    Args:
+        id: Id of Kebab button
+        locator: Kebab button locator
+        index: If nothing specified, the first kebab will be used else you can specify the number
+    """
+
+    ROOT = ParametrizedLocator("{@locator}")
+    BASE_LOCATOR = ".//div[contains(@class, 'dropdown-kebab-pf') and ./button[@id={}]]"
+    UL = './ul[contains(@class, "dropdown-menu")]'
+    BUTTON = "./button"
+    ITEM = "./ul/li/a[normalize-space(.)={}]"
+    ITEMS = "./ul/li/a"
+
+    def __init__(self, parent, id=None, locator=None, index=1, logger=None):
```

Comment:
I don't think this behavior should exist in this widget definition. providing a broad 'locator' parameter covers this use case, but I don't want to encourage that use.

Reply:
The DOM could change, with kebabs removed or inserted, breaking the expectations of the locator defined by index.  There would be no way to tell you're interacting with a different widget than the last one.